ImageManager/ImageManager.py

```diff
@@ -16,13 +17,31 @@
 
 from app import app
 
+import uuid
+from threading import Timer
+
 image = Blueprint('image', __name__)
 
 LOGGER = logging.getLogger('image-manager.' + __name__)
 LOGGER.addHandler(logging.StreamHandler())
-LOGGER.setLevel(logging.INFO)
+LOGGER.setLevel(logging.DEBUG)
+
+# global_imageid = 20
+
+
+UPLOAD_TIMEOUT = 5  # seconds
+
 
-global_imageid = 20
+def confirm_image_uploaded(imageid):
+    LOGGER.debug("Testing if image {} was uploaded".format(imageid))
+    orm_image = assert_image_exists(str(imageid))
+    data = image_schema.dump(orm_image).data
+    if data['confirmed']:
+        LOGGER.debug("Image {} was uploaded, metadata confirmed".format(imageid))
+        return
+    LOGGER.info("Image {} not uploaded, metadata discarded".format(imageid))
+    db.session.delete(orm_image)
```

Comment:
perhaps instead of removing the metadata information from the system, a better approach would be to simply display the version as not having a valid image associated with it. That way we allow a user to have software version management configuration set in the platform, even if the images themselves are not to be delivered through dojot.